jmxfetch.py

```diff
@@ -347,14 +349,19 @@ def start(cls, confd_path, agentConfig, logging_config, path_to_java, java_run_o
                 command, # Name of the command
             ]
 
-
             subprocess_args.insert(4, '--check')
             for check in jmx_checks:
                 subprocess_args.insert(5, check)
 
-            if java_run_opts:
-                for opt in java_run_opts.split():
-                    subprocess_args.insert(1,opt)
+            # Specify a maximum memory allocation pool for the JVM
+            if "Xmx" not in java_run_opts:
```

Comment:
Watchout this won't work in the (rare) case where people are using 
```java
XX:MaxHeapSize
```
and 
```java
XX:InitialHeapSize
```

to set these values